cove/templates/base_360.html

```diff
@@ -14,6 +14,17 @@
   Upload, paste or provide a link to data in the 360 Giving Data Standard format, and this application will convert between JSON, Excel and CSV formats, allowing you to download the original file, and the converted versions.{% endblocktrans %}</p>
   
   <p>{% blocktrans %}You will also be able to inspect key information from the data that the application is able to find, so that you can check the data.{% endblocktrans %}</p>
+  
+  <h3><small>{% blocktrans %}Formats{% endblocktrans %}</small></h3>
+  <p>{% blocktrans %}
+  The application accepts data in some of the formats given in the <a href="http://docs.threesixtygiving.org/publish/">360 Giving Data Standard publishing guidence</a> under <a href="http://docs.threesixtygiving.org/publish/#toc1">Choose your template</a>.
+  <br>Acceptable files are: {% endblocktrans %}</p>
+  <ul>
+    <li>{% blocktrans %}Summary Spreadsheet - <a href="http://docs.threesixtygiving.org/assets/standard/schema/summary-table/360-giving-schema-titles.xlsx">Excel</a>{% endblocktrans %}</li>
+    <!--<li>{% blocktrans %}Summary Spreadsheet - <a href="http://docs.threesixtygiving.org/assets/standard/schema/summary-table/360-giving-schema-titles.csv/Activity.csv">CSV</a>{% endblocktrans %}</li>-->
+    <li>{% blocktrans %}JSON built to the <a href="http://docs.threesixtygiving.org/docs/#json-schema">360 Giving Data Standard JSON schema</a>{% endblocktrans %}</li>
+    <li>{% blocktrans %}Multi-table data package - <a href="http://docs.threesixtygiving.org/assets/standard/schema/multi-table/360-giving-schema-fields.xlsx">Excel</a>{% endblocktrans %}</li>
```

Comment:
This is based on and replicates the text used on the 360 documentation which is why I have chosen to use them.